examples/common/python/connectors/direct/work_order_jrpc_impl.py

```diff
@@ -151,36 +152,37 @@ def __validate_data_format(self, params, in_data, out_data):
                 return False, \
                     "Invalid base64 format of in data"
         
-        for data in out_data:
-            if "dataHash" in out_data:
-                data_hash = data["dataHash"]
-                if not is_hex(data_hash) and data_hash != "":
-                    logging.error("Invalid data hash of out data")
+        if out_data is not None:
+            for data in out_data:
```

Comment:
This code is duplicated for inData and outData validation. Could be moved to a function

Reply:
Fixed